test/test_static_join/test_sj_transformer.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from typing import Optional
+
+import pytest
+
+from flynt.static_join.transformer import transform_join
+
+pytestmark = pytest.mark.skipif(
+    sys.version_info < (3, 8), reason="requires python3.8 or higher"
+)
+
+CASES = [
```

Comment:
Previously a lot of edge cases shown up when using more diverse operations. Can you add:
* indexing (elements[123])
* function calls
* list / dict comprehensions
* lambda functions

I don't care so much if they are converted or not, but they should not crash :)

Reply:
Done deal.